ntbs-service/Pages/ValidationModel.cs

```diff
@@ -55,14 +61,9 @@ protected ContentResult ValidateDate(object model, string key, string day, strin
 
         private ContentResult GetValidationResult(object model, string key)
         {
-            if (TryValidateModel(model))
-            {
-                return ValidContent();
-            }
-            else
-            {
-                return Content(ModelState[key].Errors[0].ErrorMessage);
-            }
+            TryValidateModel(model);
+
+            return ModelState[key] == null ? ValidContent() : Content(ModelState[key].Errors[0].ErrorMessage);

```

Comment:
I agree with the change to checking the individual model state only, but I quite liked this being an `if-else` statement - seems a bit more explicit in intent?